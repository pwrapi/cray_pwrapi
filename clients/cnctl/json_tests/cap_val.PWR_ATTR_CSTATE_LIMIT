{
 "PWR_Function"          : "PWR_ObjAttrValueCapabilities",
 "PWR_Attrs"             : [
                            { "PWR_AttrName" : "PWR_ATTR_CSTATE_LIMIT" }
                           ],
 "PWR_MajorVersion"      : 32,
 "PWR_MinorVersion"      : 67
}
