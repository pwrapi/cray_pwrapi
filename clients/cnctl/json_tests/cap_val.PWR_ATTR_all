{
 "PWR_Function"          : "PWR_ObjAttrValueCapabilities",
 "PWR_Attrs"             : [
                            { "PWR_AttrName" : "PWR_ATTR_CSTATE_LIMIT" },
                            { "PWR_AttrName" : "PWR_ATTR_FREQ" },
                            { "PWR_AttrName" : "PWR_ATTR_FREQ_LIMIT_MAX" },
                            { "PWR_AttrName" : "PWR_ATTR_FREQ_LIMIT_MIN" }
                           ],
 "PWR_MajorVersion"      : 32,
 "PWR_MinorVersion"      : 67
}
