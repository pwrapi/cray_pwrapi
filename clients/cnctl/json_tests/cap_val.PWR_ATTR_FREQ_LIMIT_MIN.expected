{
 "PWR_Attrs"             : [
                            {
                             "PWR_AttrName"              : "PWR_ATTR_FREQ_LIMIT_MIN",
                             "PWR_AttrValueCapabilities" : [
                                                            2301000,
                                                            2300000,
                                                            2200000,
                                                            2100000,
                                                            2000000,
                                                            1900000,
                                                            1800000,
                                                            1700000,
                                                            1600000,
                                                            1500000,
                                                            1400000,
                                                            1300000,
                                                            1200000
                                                           ],
                             "PWR_ReturnCode"            : 0
                            }
                           ],
 "PWR_MajorVersion"      : 0,
 "PWR_MinorVersion"      : 1,
 "PWR_ReturnCode"        : 0,
 "PWR_Messages"          : null,
 "PWR_ErrorMessages"     : null
}

